aspnetcore/performance/caching/middleware.md

```diff
@@ -178,6 +180,10 @@ The sample app adds headers to control caching on subsequent requests:
 
 [!code-csharp[](middleware/samples_snippets/2.x/AddHeaders.cs)]
 
+The preceding headers are not written to the response and are overriden when a controller, action, or Razor Page:
+
+* Has a [[ResponseCache]](xref:Microsoft.AspNetCore.Mvc.ResponseCacheAttribute) attribute. Setting the `[ResponseCache]` attribute to contain [VaryByHeader](/aspnet/core/performance/caching/response#vary) doesn't change this behavior.
```

Comment:
I'm a little confused by what you mean by `doesn't change this behavior.` It seem contradictory.

Reply:
I'm hesitant to mix in once this has passed to internal Microsoft review, but in lieu of knowing for sure, I pushed an update to the branch.

Now the bullet point reads
> Has a [[ResponseCache]](xref:Microsoft.AspNetCore.Mvc.ResponseCacheAttribute) attribute. This applies even if a property isn't set. For example, omitting the [VaryByHeader] property (/aspnet/core/performance/caching/response#vary) will cause the corresponding header to be removed from the response.

Reply:
Also, this is only in ::: moniker range="< aspnetcore-3.0", but should also be in the moniker range=">= aspnetcore-3.0"

I haven't made the change yet, because it'll just be annoying updating 2 places untill the text passes review.

(Probably that was the reason it was omitted until now.)